kendo/angular-v-react/article.md

```diff
@@ -44,13 +42,14 @@ src
 │   ├── app.component.ts
 │   ├── app.module.ts
 │   └── helloworld
-│       ├── helloworld-routing.component.ts
```

Comment:
I don't think there would be a routing file here for a hello world app.

Reply:
Good point. I’ll note that this is a hello world app with a routing structure in place, or something like that.